pkg/oci/options.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2025 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package oci
+
+const (
+	// MediaTypePipedPlugin is the media type for PipeCD Agent plugins.
+	MediaTypePipedPlugin = "application/vnd.pipecd.piped.plugin"
+)
```

Comment:
@Warashi 
[ask] Is `pkg/oci` expected to be used for the other case, except for downloading the plugin?
If not, how about adding it under the `pkg/plugin`?

Reply:
@ffjlabo 
I want to clarify what we should place under the package/plugin. I think we should put the packages used to implement the plugin and only them. If we place features that are not used to implement the plugin, the plugin developers may be confused about which package to use. WDYT?

Reply:
@Warashi 
> I think we should put the packages used to implement the plugin and only them.
I got your point. Sorry, I was told it some time before.

On the other hand, I think it would be nice to minimize the package's scope to avoid using it for an unexpected use case. How about in `pkg/app/pipedv1/plugin`?

Reply:
@Warashi 
> I want to implement a pipectl subcommand to push plugins to the OCI Registry, so I want to use this in pipectl. So, I don't want to make this scope to pipedv1.

I see, surely that would be strange, package pipectl imports pipedv1's sub-package.
OK, let's keep it as is.